election-api-scala/app/model/PartyResult.scala

```diff
@@ -0,0 +1,9 @@
+package model
+
+case class PartyResult(party: String, votes: Int, share: BigDecimal)
+case class ConstituencyResult(id: Int, name: String, seqNo: Int, partyResults: Seq[PartyResult])
+
+case class ApiResponse (error: String, message: String)
+
+
+case class Scoreboard(declared:Int)
```

Comment:
I know this might be nit-pick, but the go-lang version doesn't define a struct for Scoreboard. I think that we should get rid of `declared`

Reply:
Minor problem, can only declare a case class if it has a property. So its just a language constraint.

Reply:
Fair enough